### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by lfmnovaes**
- Removed `current_user` method in `ApplicationController`
- Installed and setup devise
  - Can register a new user
  - User logs in with a combination of email and password
  - Hashed password should be stored in the database
  - Ask for confirmation of email
  - Can reset password
- Modified the views in devise for registration and login to match the wireframes from the Sneak Peek and your styling

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  </div>
+
+  <% if @posts_counter.nil? %>
+    <%= f.hidden_field :posts_counter, value: 0 %>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```
Comment (ElijahTobs) [inline app/views/devise/registrations/new.html.erb:28]: - [ ]  Your application is supposed to allow for registering a new user but I was unable to register a new user.

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi Team 👋

You've done really well so far! 👍 

## Major Highlights 💯 

README is professional ✔️ 
You've followed the Gitflow rules. ✔️ 
Good commit messages ✔️ 
No linter errors ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️




_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

Check for changes marked as Optional in the inline comments below.


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (lfmnovaes) [reply app/views/devise/registrations/new.html.erb:28]: Did you click on the confirmation link in your terminal/console?
I've just run `rails db:reset` and tried to register a new user and it's working just fine
image

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @lfmnovaes 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — COMMENTED** [review]
Hello  @lfmnovaes 👋,

_This is an additional review as a result of the second opinion request._

I was able to create a new user, but I'd say that the added steps in the readme file are really helpful. Glad that you got approved already.

Keep rocking 🚀!

**Pull request merged**